supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -27,3 +27,30 @@ Provide the source language if it is supported by the Red Hat Customer Portal to
 == Long code examples in procedures
 
 All code blocks (regardless of length) must be necessary, accurate, and helpful. Code blocks must be as copy-and-paste friendly as possible, with the exception of variables and callouts. The length of the block is irrelevant, within reason, if the code block follows these guidelines.
+
+[[command-line-commands-that-require-elevated-privileges]]
+== Command-line commands that require elevated privileges
+
+Some command-line interface (CLI) commands require that a user run them with "elevated privileges".  
+That is, some commands require that a normal user switch to another user account whose privileges permit certain operations or require that a normal user temporarily change their current privileges without changing user accounts.
```

Comment:
The term "normal" might be hard to gauge. An advanced user might think they're normal. Maybe define a normal user? How about "regular user account"?

Maybe both use-case scenarios could be bulleted?